components/api/db/cellery_hub_db.bal

```diff
@@ -20,12 +20,23 @@ import ballerina/log;
 import ballerina/mysql;
 import ballerina/sql;
 import cellery_hub_api/gen;
+import cellery_hub_api/constants;
 import ballerina/io;
 import ballerina/encoding;
 
+type RegistryOrgUserMapping record {|
+    string USER_ROLE;
+|};
+
+type RegistryArtifactImage record {|
+    string ARTIFACT_IMAGE_ID;
+|};
+
 public function getOrganization(string orgName) returns json | error {
     log:printDebug(io:sprintf("Performing data retreival on REGISTRY_ORGANIZATION table, Org name : \'%s\': ", orgName));
-    table<record {}> res =  check connection->select(GET_ORG_QUERY, gen:OrgResponse, orgName, loadToMemory = true);
+    table< record {
+
+}> res = check connection->select(GET_ORG_QUERY, gen:OrgResponse, orgName, loadToMemory = true);
```

Comment:
Shall we fix these formatting issues ? These seems too much to ignore till Ballerina tooling formatting fixes come.